slashing/program/src/processor.rs

```diff
@@ -66,3 +73,86 @@ pub fn process_instruction(
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use {
+        super::verify_proof_data,
+        crate::{
+            duplicate_block_proof::DuplicateBlockProofData, error::SlashingError,
+            shred::tests::new_rand_data_shred,
+        },
+        rand::Rng,
+        solana_ledger::{blockstore_meta::DuplicateSlotProof, shred::Shredder},
+        solana_sdk::{
+            clock::{Clock, Slot, DEFAULT_SLOTS_PER_EPOCH},
+            program_error::ProgramError,
+            signature::Keypair,
+            signer::Signer,
+        },
+        std::sync::Arc,
+    };
+
+    const SLOT: Slot = 53084024;
+    static mut CLOCK_SLOT: Slot = SLOT;
```

Comment:
This is fine for just one test, but in case we need to, we can do proper locking on this variable, like at https://github.com/solana-labs/solana-program-library/blob/4b914915f45054c7fbc3be42c57031a1eaf7cb89/token/program/src/processor.rs#L1060-L1063

Reply:
nice ya that's less hacky 5210f9cee99d136f7061f453c7efa171a09a4166